keras_cv/models/segmentation/deeplab.py

```diff
@@ -13,9 +13,13 @@
 # limitations under the License.
 
 import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
 
 from keras_cv.layers.spatial_pyramid import SpatialPyramidPooling
+from keras_cv.models import utils
 from keras_cv.models.resnet_v2 import ResNetV2
+from keras_cv.models.segmentation.__internal__ import SegmentationHead
```

Comment:
See comment below, we should simply remove the __internal__ implementation

Reply:
Removed